scrapy/core/downloader/handlers/http11.py

```diff
@@ -121,8 +121,9 @@ class TunnelingTCP4ClientEndpoint(TCP4ClientEndpoint):
     with this endpoint comes from the pool and a CONNECT has already been issued
     for it.
     """
-
-    _responseMatcher = re.compile(br'HTTP/1\.. (?P<status>\d{3})(?P<reason>.{,32})')
+    _truncatedLength = 1000
+    _responseAnswer = 'HTTP/1\\.. (?P<status>\\d{3})(?P<reason>.{,' + str(_truncatedLength) + '})'
+    _responseMatcher = re.compile(fr'{_responseAnswer}'.encode('utf-8'))
```

Comment:
This should be the same:

```suggestion
    _responseMatcher = re.compile(_responseAnswer.encode())
```